docs/conf.py

```diff
@@ -31,7 +31,8 @@
     "notmuch",
     "requests",
     "bs4",
-    "novaclient.v2"
+    "novaclient.v2",
+    "psutil", "getpass"
```

Comment:
- getpass is part of the Python standard library
- psutil is already on the list (line 25)